website/pages/en/sps/triggers.mdx

```diff
@@ -0,0 +1,11 @@
+----
```

Comment:
```suggestion
---
```
This extra dash breaks the layout of the page:

![CleanShot 2024-09-13 at 15 23 20@2x](https://github.com/user-attachments/assets/6c13e460-8488-43bf-8753-5e4899423da2)